NMS-15215 - Number examples in service monitor chapters (#5779)
OpenNMS/opennms: docs/modules/reference/pages/service-assurance/monitors/FtpMonitor.adoc

> Example configuration on how to configure the monitor in `poller-configuration.xml`.

Reviewer: ```suggestion
This example demonstrates how to configure the monitor in `poller-configuration.xml`:
```

  <parameter key="timeout" value="3000"/> <2>
  <parameter key="port" value="21"/> <3>
  <parameter key="userid" value=""/> <4>
  <parameter key="password" value=""/> <5>
</service>

<service name="FTP-With-Auth-From-Asset" interval="300000" user-defined="false" status="on">
  <parameter key="retry" value="1"/> <1>
  <parameter key="timeout" value="3000"/> <2>
  <parameter key="port" value="21"/> <3>
  <parameter key="userid" value="{username}"/> <4>
  <parameter key="password" value="{password}"/> <5>
</service>

<service name="FTP-Customer" interval="300000" user-defined="false" status="on">
  <parameter key="retry" value="1"/> <1>
  <parameter key="timeout" value="3000"/> <2>
  <parameter key="port" value="21"/> <3>
  <parameter key="userid" value="Customer"/> <4>
  <parameter key="password" value="MySecretPassword"/> <5>
</service>

<monitor service="FTP" class-name="org.opennms.netmgt.poller.monitors.FtpMonitor"/> <6>
<monitor service="FTP-With-Auth-From-Asset" class-name="org.opennms.netmgt.poller.monitors.FtpMonitor"/> <6>
<monitor service="FTP-Customer" class-name="org.opennms.netmgt.poller.monitors.FtpMonitor"/> <6>
----
<1> Number of attempts to test a service’s status

> <2> Timeout for the isReachable method, in milliseconds.

Author: Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.
